Wine_project_Tanya_new.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>spelling</p><p> </p><p>Explain why have you mentioned this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-TataAndBigData/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>